clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ClusterController.groovy

```diff
@@ -100,7 +101,7 @@ class ClusterController {
     }
     clusters.removeAll([null])
     if (!clusters) {
-      throw new ClusterNotFoundException(cluster: name)
+      throw new NotFoundException("Cluster not found (application: ${application}, account: ${account}, name: ${name})")
```

Comment:
Couldn't really think of a good reason to use explicit exceptions (vs the generic `NotFoundException`) inside of a controller.